src/test/java/ua/biz/test/tests/vacancy/VacancyDataTest.java

```diff
@@ -0,0 +1,43 @@
+package ua.biz.test.tests.vacancy;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import ua.biz.test.base.BaseTest;
+import ua.biz.test.entity.Vacancy;
+import ua.biz.test.pages.CreateVacancyPage;
+import ua.biz.test.pages.VacanciesListPage;
+
+import java.util.List;
+
+public class VacancyDataTest extends BaseTest {
+
+    private Vacancy TEST_VACANCY = new Vacancy("Manager", "Ivan Mosiychuk", 2, true);
+
```

Comment:
Why variable name `TEST_VACANCY ` is all capital letter. Only constants style respond to such option.

Reply:
It must be final. Fixed.